build/Makefile.openssl

```diff
@@ -7,6 +7,6 @@ $(INSTALL_DIR)/bin/openssl: Makefile
 
 Makefile: config Configure Makefile.org
 	# Force a 64 bit build on MacOS
-	KERNEL_BITS=64 ./config --openssldir=$(INSTALL_DIR) no-shared enable-static-engine -fPIC
+	KERNEL_BITS=64 ./config --install_prefix=$(INSTALL_DIR) --prefix=/ --libdir=lib --openssldir=/etc/ssl no-shared enable-static-engine -fPIC
```

Comment:
Sorry, in case that wasn't clear. I meant we're OK with people having to specify the path if they want to run a prod log?